torch/_functorch/_aot_autograd/input_output_analysis.py

```diff
@@ -377,9 +377,10 @@ def create_graph_signature(
     graph_output_names = _graph_output_names(fx_g)
 
     num_params_buffers = len(param_names) + len(buffer_names)
+    num_tokens = len(fw_metadata.tokens)
```

Comment:
Did you want to test multiple types of tokens or should we punt that? I'm fine with just punting it (to test multiple types you'd need to define another op that has a different side effect :P. Maybe "_print(str, ordered=True)" ?)

Reply:
It's good to test it now so that we don't specialize on num_tokens=1 in the followup work (export+inductor)

Reply:
Does that mean I should create an "OrderedEffect" and "UnorderedEffect" where if print(...ordered=True) then it's an OrderedEffect otherwise it's UnorderedEffect?

Reply:
Er, maybe punt this until we have a use case. Any sort of "unordered effect" needs to be implemented slightly differently, that is, each call to print(ordered=False) needs to get its own token.

Reply:
if we want to not bother with tests for now / punt, maybe add a comment / assert somewhere for now that `num_tokens == 1`?